prov/gni/src/gnix_cm.c

```diff
@@ -86,8 +86,8 @@ static int gnix_getname(fid_t fid, void *addr, size_t *addrlen)
 	/*
 	 * Retrieve the cdm_id & device_addr from the gnix_nic structure.
 	 */
```

Comment:
I think I might be confused here.  Will `gnix_getname` only be called with endpoints of type FI_EP_RDM?  Even if the answer is yes, we should add an `assert(ep->cm_nic)` here.